server/lib/pdf.js

```diff
@@ -13,7 +13,8 @@ export const getTransactionPdf = async (transaction, user) => {
   const headers = {
     Authorization: `Bearer ${accessToken}`,
   };
-  return await fetch(pdfUrl, { method: 'get', headers })
+
+  return fetchWithTimeout(pdfUrl, { method: 'get', headers })
```

Comment:
Maybe we could pass a different timeout here? PDF service can sometimes be slow, I think 10 seconds would be a good start.